Common/Net/HTTPClient.h

```diff
@@ -202,6 +202,7 @@ class Downloader {
 	std::shared_ptr<Download> AsyncPostWithCallback(
 		const std::string &url,
 		const std::string &postData,
+		const std::string &postMime, // Use postMime = "application/x-www-form-urlencoded" for standard form-style posts, such as used by retroachievements.
```

Comment:
This one only seems to only need simple text data (so far), and it's already being encoded for us by the rcheevos library - so I don't think bringing in MultipartFormDataEncoder here makes sense, since we're just forwarding already-encoded data. Simply adding the postMime parameter here was the easy way to go.

But yes, as you state in the other comment we're approaching DownloadDesc territory.

I'll clean up the comment.